### DISCUSSION TITLE
Mistake in documentation

### DISCUSSION THREAD

**Discussion by MrKozelberg** [NCAR/noahmp#152] [Q&A]
There is a mistake in your [technical note](https://github.com/NCAR/noahmp/blob/fa2d2098bac1e437a21ccc15494b152996d5bc84/docs/NoahMP_v5_technote.pdf) in equations where you calculate $F_{MO,tmp}$ (Eq.3.6.190, Eq. 3.6.362). The actually way of this variable limitation little differs and writes as follows:
`       TMP1 = ConstVonKarman * (ConstGravityAcc/TVIR) * HeatSensibleTmp / (DensityAirRefHeight*ConstHeatCapacAir)`
`       if ( abs(TMP1) <= MPE ) TMP1 = MPE`

**Comment by cenlinhe** [comment]
Thank you for pointing out this typo in the technical note. Yes, they are indeed slightly different and should be corrected in the technical note. We will update this in the next version of the technical note.
